efi/pcr_profile.go

```diff
@@ -28,15 +28,36 @@ import (
 	"golang.org/x/xerrors"
 )
 
+// PCRProfileEnablePCRsOption is an option for AddPCRProfile that adds one or more PCRs.
+type PCRProfileEnablePCRsOption interface {
+	PCRProfileOption
+	PCRs() tpm2.HandleList
+}
+
```

Comment:
nitpick: shouldn't this definition come after the PCRProfileOption one?

Reply:
I've moved this now